lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
+
+1. MAJOR när du gör inkompatibla API-förändringar,
+1. MINOR när du lägger till funktionalitet på ett bakåtkompatibelt sätt, och
+1. PATCH när du gör bakåtkompatibla buggfixar.
+
+Ytterliga notationer finns tillgängliga som utökning till MAJOR.MINOR.PATCH-formatet 
```

Comment:
Beteckning, va? As in, betäckning = sexy time for a mare, beteckning = designation.